ethereum/datastructures/src/main/java/tech/pegasys/artemis/datastructures/util/AttestationUtil.java

```diff
@@ -82,8 +82,15 @@ public static boolean verifyAttestation(BeaconState state, Attestation attestati
       BeaconState headState,
       HashMap<UnsignedLong, List<Triple<List<Integer>, UnsignedLong, Integer>>>
           committeeAssignments) {
-
     UnsignedLong slot = headState.getSlot();
+    return getAttesterInformation(headState, committeeAssignments, slot);
+  }
+
+  public static List<Triple<BLSPublicKey, Integer, CrosslinkCommittee>> getAttesterInformation(
+      BeaconState state,
+      HashMap<UnsignedLong, List<Triple<List<Integer>, UnsignedLong, Integer>>>
+          committeeAssignments,
+      final UnsignedLong slot) {
```

Comment:
Broke up this method into 2 methods so we could ask for attester info at a specific slot.